app/react/src/client/preview/utils.ts

```diff
@@ -1,6 +1,27 @@
+/* eslint-disable no-underscore-dangle */
+import global from 'global';
 import { defaultDecorateStory, combineParameters } from '@storybook/client-api';
 import { GlobalConfig, Meta, Story, StoryContext } from './types-6-0';
 
+/** Function that sets the globalConfig of your storybook. The global config is the preview module of your .storybook folder.
+ *
+ * It should be run a single time, so that your global config (e.g. decorators) is applied to your stories when using `composeStories` or `composeStory`.
+ *
+ * Example:
+ *```jsx
+ * // setup.js (for jest)
+ * import { setGlobalConfig } from '@storybook/react';
+ * import * as globalStorybookConfig from './.storybook/preview';
+ *
+ * setGlobalConfig(globalStorybookConfig);
+ *```
+ *
+ * @param config - e.g. (import * as globalConfig from '../.storybook/preview')
+ */
+export function setGlobalConfig(config: GlobalConfig) {
+  global.__STORYBOOK__GLOBAL__CONFIG__ = config;
```

Comment:
Could this just be a local variable to this file?

Reply:
Great idea. done!